Documentation/Upgrade/README.rst

```diff
@@ -0,0 +1,106 @@
+.. _README:
+
+.. include:: ../lib/doc-includes/pike-icons.inc
+
+.. Don't use default python highlighting for code blocks http://www.sphinx-doc.org/en/stable/markup/code.html
+
+.. highlight:: none
+
+
+===================================
+Minor Upgrade to Release 5.4.1 U8+
+===================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Use this documentation when upgrading between minor releases. The process applies to upgrades from Release 5.4.1 U5 to Release 5.4.1 U8+. During this process, the Nuage components and Red Hat can be updated at the same time.
+
+It is assumed the operator is familiar with OpenStack upgrades, VSP installation, the distribution-specific installation and upgrade practices, and the specific requirements for operations in a production environment.
+
+
+Upgrade Paths
+-------------
+
+Upgrading to the Release 5.4.1 U8+ is supported only on OSPD-based deployments. In this release, you can upgrade only from Queens 5.4.1 U5 to Queens 5.4.1 U8.

```

Comment:
This first sentence puzzles me . Also outside OSPD we support upgrade.

Reply:
done - removed the first sentence